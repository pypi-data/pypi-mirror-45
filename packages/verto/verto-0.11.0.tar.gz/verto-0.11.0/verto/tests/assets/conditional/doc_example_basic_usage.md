{conditional if condition="version == 'teacher'"}

This is text that only teachers should see.

{conditional end}
