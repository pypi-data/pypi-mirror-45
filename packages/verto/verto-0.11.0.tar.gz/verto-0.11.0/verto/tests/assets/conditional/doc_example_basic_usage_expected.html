{% if version == 'teacher' %}
<p>This is text that only teachers should see.</p>
{% endif %}
